lz4/stream/__init__.py

```diff
@@ -0,0 +1,207 @@
+from ._stream import _LZ4StreamContext, _compress, _decompress, _get_block
+from ._stream import LZ4StreamError, _compress_bound, _input_bound, LZ4_MAX_INPUT_SIZE  # noqa: F401
+
+
+__doc__ = """\
+A Python wrapper for the LZ4 stream protocol.
+"""
+
+# Basic sample::
+#
+#   from lz4.stream import LZ4StreamCompressor, LZ4StreamDecompressor
+#
+#   # LZ4 stream context properties:
+#   block_size_length = 2   # LZ4 compressed block size stored on 2 bytes
+#   page_size         = 256 # LZ4 context double buffer page size
+#
+#   origin_stream       = ... # from somewhere
+#   compressed_stream   = bytearray()
+#   decompressed_stream = bytearray()
+#
+#   # LZ4 stream compression of origin_stream into compressed_stream:
+#   with LZ4StreamCompressor(page_size, store_comp_size=block_size_length) as proc:
+#       offset = 0
+#       while offset < len(origin_stream):
+#           chunk = origin_stream[offset:offset + page_size]
+#           block = proc.compress(chunk)
+#           compressed_stream.extend(block)
+#           offset += page_size
+#
+#   # LZ4 stream decompression of compressed_stream into decompressed_stream:
+#   with LZ4StreamDecompressor(page_size, store_comp_size=block_size_length) as proc:
+#       offset = 0
+#       while offset < len(compressed_stream):
+#           block = proc.get_block(compressed_stream[offset:])
+#           chunk = proc.decompress(block)
+#           decompressed_stream.extend(chunk)
+#           offset += block_size_length + len(block)
+
+
+class LZ4StreamDecompressor(_LZ4StreamContext):
+    """ LZ4 stream decompressing context.
+
+    """
+    def __init__(self, page_size, return_bytearray=False, store_comp_size=4, dictionary=""):
+        """ Instantiates and initializes a LZ4 stream decompression context.
+            Raises an exception if any error occurs.
```

Comment:
Multiline docstrings MUST begin with a single line summary and then a blank line, as they're used by indexing tools etc.

https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings